styles/shared/BackToSearchResults.scss

```diff
@@ -0,0 +1,16 @@
+.item-title {
+    display: flex;
+    flex-direction: column;
+
+    .back-to-search-link {
+        line-height: calc(1.1 * var(--font-size-4));
+    }
+
+    @media (min-width: $breakpoint-medium) {
+        flex-direction: row;
+        .back-to-search-link {
+            padding-left: var(--space-s);
+            min-width: 190px;
```

Comment:
nit: where are you getting this value?  In general we like to use ems instead of px as ems are relative to font size.  It's not a big deal right now, as px and em and everything are being used interchangeably in the app, but it's something to keep in mind as we move to new DS.